File: 2243-check-if-all-as-appears-before-all-bs/check-if-all-as-appears-before-all-bs.cs
public class Solution {
    public bool CheckString(string s) {
        for(int i = 0; i < s.Length - 1; i++){
            if(s[i] == 'b'){
                while(i < s.Length){
                    if(s[i++] == 'a'){
                        return false;
                    }
                }
            }
        }
        return true;
    }
}

File: 2243-check-if-all-as-appears-before-all-bs/check-if-all-as-appears-before-all-bs.java
class Solution {
    public boolean checkString(String s) {
        for(int i = 0; i < s.length() - 1; i++){
            if(s.charAt(i) == 'b'){
                while(i < s.length()){
                    if(s.charAt(i++) == 'a'){
                        return false;
                    }
                }
            }
        }
        return true;
    }
}